### DISCUSSION TITLE
How to toggle to light mode as like the original settings of site?

### DISCUSSION THREAD

**Discussion by Archie-2021** [darkreader/darkreader#9247] [General]
It used to be that toggle just changed the effect on and off from the current tab, now it just change it to a different shade of dark mode. i don't want to do change Darkmode setting for most sites permanently. just want to disable it temporarily. How i can do that?

**Comment by Gusted** [comment]
The site toggle still has the same effect, are you using the correct button?

**Comment by Archie-2021** [comment]
![image](https://user-images.githubusercontent.com/85870778/178280167-990c511f-1096-439e-ad59-1dcaf0cfb793.png)
i am toggling both dark/light and on/off button and it only changes the shades of the dark. i suppose it should be a bug or something then. i am using Brave on Windows 10

**Reply by Gusted** [reply]
Have you noticed the button in the top left corner? Please try to use that corner, you're currently not using the site toggle. 

The "light" option means that Dark Reader is still active, but applies a light theme over the website.

**Reply by Archie-2021** [reply]
> Have you noticed the button in the top left corner? Please try to use that corner, you're currently not using the site toggle.
> 
> The "light" option means that Dark Reader is still active, but applies a light theme over the website.

yes i tried that check mark too, forgot to mention it. there is just handful of options in this plugin and i think i have tried every thing without any results

**Reply by Gusted** [reply]
Could you make a screenshot of the site when you have the site disabled(so top-left button) and note how it differ from the original site.

**Comment by Archie-2021** [comment]
there is a difference but it is very small, hope to be detectable for you on screenshots

On:
![image](https://user-images.githubusercontent.com/85870778/178499145-601a2341-1de6-4d65-bf6f-f3f6742439f4.png)
Off:
![image](https://user-images.githubusercontent.com/85870778/178499203-6054600f-4ab6-4acc-8bb8-a886336a4e93.png)
settings:
![image](https://user-images.githubusercontent.com/85870778/178499299-2e05a611-4939-4b52-914b-edbbbd7e99b6.png)

**Reply by Gusted** [reply]
You might have Github's dark mode enabled: https://github.com/settings/appearance

**Reply by Archie-2021** [reply]
> You might have Github's dark mode enabled: https://github.com/settings/appearance

I tested on google and m some other websites too at the start. but now you said it I tested it on more websites and only on Reddit and some other ones it works. I think you are right, but it is weird because i can't find and appearance option like what you linked for github in most of them , I guess they just use mostly system theme by default. maybe you can add a real light-mode too. so having a dark mode, a default/null mode and light mode. Just as an idea.

thank you very much for your help and this awesome plugin
